skyportal/handlers/api/photometry.py

```diff
@@ -181,6 +186,7 @@ def post(self):
 
             phot.original_user_data = packet
             phot.bulk_upload_id = bulk_upload_id
```

Comment:
It looks like `bulk_upload_id` can be `None` based on the logic above. This will make the id non-unique and so deleting by this id will cause the removal of unrelated photometry. Perhaps you could fallback to hash of the datetime string with a random value appended?

Reply:
I'm changing this to `upload_id` and it will always be non-null